src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -81,4 +72,16 @@ Blank()
 "B"
 
 >> Char(1/0)
-#Error
\ No newline at end of file
+#Error
+
+>> Text(DateTimeValue("03/15/2018 12:00:00"), "yyyy-MM-ddTHH:mm:ss")
+"2018-03-15T12:00:00"
+
+>> Text(DateValue("03/15/2018 12:00:00"), "yyyy-MM-dd")
+"2018-03-15"
+
+>> Text(DateValue("03/15/2018"), "yyyy-MM-dd")
```

Comment:
It is better to use `Date(2018, 3, 15)` instead of `DateValue("03/15/2018")` since the former works in all regions, and the latter won't work in regions where the date format is dd/mm/yyyy.

Reply:
Got it, thanks!